README.md

```diff
@@ -79,14 +79,41 @@ There are two types of entry points:
   * `prod` - Run the UI and API apps in "production" mode (port 8000 for UI,
     8001 for API). Note that this requires the JS be compiled already.
 1. One use commands which run until complete. These are ran via
-  `docker-compose run`
+  `docker-compose run --rm` (the `--rm` just deletes the images after running;
+  it's not strictly required)
   * `manage.py`
   * `py.test`
   * `flake8`
   * `pip-compile`
   * `npm`
   * `webpack`
 
+### Resolving common container issues
+
+If a Javascript dependency has been added (indicated by an error within
+`node_modules`), run
+```sh
+docker-compose run --rm npm install
```

Comment:
`docker-compose build` and `npm install` have to be run the first time through right? If so, should we add a short section at the beginning to mention this? Otherwise it looks like you can jump right in to the migrate/superuser stuff.